helm/happa/templates/deployment.yaml

```diff
@@ -0,0 +1,105 @@
+apiVersion: apps/v1
+kind: Deployment
```

Comment:
Why do we have all those new tamplates?

Reply:
https://github.com/giantswarm/happa/pull/1500